tests/discovery/test_hosts.py

```diff
@@ -2,40 +2,33 @@
 import time
 from queue import Empty
 
-from src.modules.discovery.hosts import FromPodHostDiscovery, RunningAsPodEvent, HostScanEvent, AzureMetadataApi
+from src.modules.discovery.aks import AzureHostDiscovery, AzureMetadataApi
+from src.modules.discovery.hosts import HostScanEvent, RunningPodOnCloud
 from src.core.events.types import Event, NewHostEvent
 from src.core.events import handler
+from src.core.types import CloudTypes
+
 from __main__ import config
 
-def test_FromPodHostDiscovery():
+def test_AzureHostDiscovery():
 
     with requests_mock.Mocker() as m:
-        e = RunningAsPodEvent()
+        e = RunningPodOnCloud(cloud=CloudTypes.AKS)
 
         config.azure = False
         config.remote = None
         config.cidr = None
         m.get("http://169.254.169.254/metadata/instance?api-version=2017-08-01", status_code=404)
-        f = FromPodHostDiscovery(e)
-        assert not f.is_azure_pod()
-        # TODO For now we don't test the traceroute discovery version
-        # f.execute()
+        f = AzureHostDiscovery(e)
+        assert not f.is_azure_api()
 
         # Test that we generate NewHostEvent for the addresses reported by the Azure Metadata API
         config.azure = True
         m.get("http://169.254.169.254/metadata/instance?api-version=2017-08-01", \
             text='{"network":{"interface":[{"ipv4":{"subnet":[{"address": "3.4.5.6", "prefix": "255.255.255.252"}]}}]}}')
-        assert f.is_azure_pod()
-        f.execute()
-
-        # Test that we don't trigger a HostScanEvent unless either config.remote or config.cidr are configured
```

Comment:
The tests that were for FromPodDiscovery were testing the azure discovery.. because its now separated, I didn't really could think of what to test there